dashboard/app/helpers/script_levels_helper.rb

```diff
@@ -36,9 +36,18 @@ def next_progression_level_or_redirect_path(script_level)
         script_level.next_progression_level
       end
 
-    next_level ?
-        build_script_level_path(next_level) :
-        script_completion_redirect(script_level.script)
+    if script_level.level.try(:is_plc_evaluation?)
```

Comment:
nit: usual ruby style would be to call this `plc_evaluation?` because is and ? are redundant (eg. `Array#empty?` not `Array#is_empty?`)